src/checks_util.cc

```diff
@@ -37,11 +37,16 @@
 
 namespace zetasql::linter {
 
+absl::string_view GetName(const ParseLocationRange &range,
+                          const absl::string_view &sql) {
+  const int &start = range.start().GetByteOffset();
```

Comment:
`const int start` and same for `end`

Reply:
Ah yes, its just integer, so pointer actually more costly. (4 vs. 8 byte)